editors/code/package.json

```diff
@@ -904,4 +913,4 @@
             ]
         }
     }
-}
+}
```

Comment:
For some reason, VS Code auto-formatter removes the trailing newline from `json` files specifically. I can manually restore this newline, but, I guess, anyone who use VS Code with enabled on-save formatting, and will be editing this file, will face the same issue.

Reply:
You need to set `"files.insertFinalNewline": true`.

Reply:
I see, thanks!